bls-runtime/src/macros.rs

```diff
@@ -0,0 +1,32 @@
+#[macro_export]
+macro_rules! plog {
+    ($level: expr, $($args:tt)+) => {{
+        crate::plog::plog($level, std::format_args!($($args)+))
+    }};
+}
+
+/// export the perror macro for log the error
+#[macro_export]
+macro_rules! perror {
+    ($($args:tt)+) => {{
+        crate::plog!(log::Level::Error, $($args)+)
+    }};
+
+    () => {{
+        use log::Level;
```

Comment:
maybe import this at top level? otherwise this may be imported every time we log.
Im not sure if compiler optimizes this away though.